dev/test-images/buildable-test-server/resources/cgi-bin/index.cgi

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+cat <<EOF
```

Comment:
does this need to be CGI? Does httpd not serve static files?

Reply:
yes, but it feels the need to add a newline and I can't set the header requiring Content-Length:12 (which removes the newline)

I could also just add a \n in the test comparison...

Reply:
that sounds good to me, I've put in a hello.txt file in this PR. I'm following this up with another pr with an env.cgi that returns some environment variable.